ignite/metrics/accuracy.py

```diff
@@ -130,12 +132,14 @@ def thresholded_output_transform(output):
     def __init__(
         self,
         output_transform: Callable = lambda x: x,
+        logits_to_label_fn: Callable = lambda x: torch.argmax(x, dim=1),
```

Comment:
Can't we use predefined functions here for each of the cases?

Reply:
I don't understand what cases you're talking about.

Reply:
below there is a case switching for different cases (`self._type`) and for each of the cases I would define a logins_to_label_fn during init instead of having a general interface (kind of same discussion as below)

Reply:
Ok I see. Cases "binary" and "multilabel" don't need logits_to_label_fn call. I understand the idea to have an identity function for these cases and `apply logits_to_label_fn` in all cases. But I am skeptical about it. The problem that you point out sounds to me design concerns.